exercises/ruby-1-b/ruby_1_b.rb

```diff
@@ -4,52 +4,12 @@ class Library
   CATALOG_DATA = <<~DATA
     The Adventures of Tom Sawyer;9780191604928;Mark Twain;2007
     Republic;9780718198916;Plato;2012
-    Programming Ruby: The Pragmatic Programmers' Guide;9780974514055;David Thomas;2004
+    Programming Ruby: The Pragmatic Programmers Guide;9780974514055;David Thomas;2004
```

Comment:
Our code editor can't cope with syntax highlighting if this is here, so I've just removed it.